src/main/avro/LandingZone.avsc

```diff
@@ -3,6 +3,14 @@
   "name": "LandingZoneModel",
   "namespace": "bio.terra.aws.resource.discovery.avro",
   "fields": [
+    {
+      "name": "app_framework_private_subnet_id",
+      "type": "string"
+    },
+    {
```

Comment:
Applies to appFramework as well as other resources - is it worth creating a sub-object for each resource type, assuming each resource type might have few custom properties, as well as similar properties with different values. 
something like - 

```
landingZoneModel {
    {
        name: infra
        type: { infra record - includes vpc id } // similar to metadata model
    },
    {
        name: s3Folder
        type: { s3Folder record - includes bucket name }
    },
   {
        name: sagemakerNotebook
        type: { sagemakerNotebook record with lifecycle arn, etc }
    },
   {
        name: appFramework
        type: { appFrameworkModel record with subnet id, etc}
    }
}
```

Reply:
This must match the Terraform output JSON exactly.  This would require refactoring our TF module outputs and a breaking release, so not sure it gives enough value for that.